Makefile

```diff
@@ -0,0 +1,167 @@
+ci:	## Run in automation environment
+	$(eval DISABLE_PASSCODE=true)
+	$(eval AUTO_APPROVE=-auto-approve)
+	$(eval SP_AUTH=true)
+
+set-azure-resource-group-tags: ##Tags that will be added to resource group on it's creation in ARM template
+	$(eval RG_TAGS=$(shell echo '{"Portfolio": "Early Years and Schools Group", "Parent Business":"Teaching Regulation Agency", "Product" : "Access Your Teaching Certificates", "Service Line": "Teaching Workforce", "Service": "Teacher Training and Qualifications", "Service Offering": "Access Your Teaching Certificates", "Environment" : "$(ENV_TAG)"}' | jq . ))
+
+set-azure-template-tag:
+	$(eval ARM_TEMPLATE_TAG=1.1.1)
+
+.PHONY: read-keyvault-config
+read-keyvault-config:
+	$(eval KEY_VAULT_NAME=$(shell jq -r '.key_vault_name' terraform/workspace_variables/$(DEPLOY_ENV).tfvars.json))
+	$(eval KEY_VAULT_SECRET_NAME=INFRASTRUCTURE)
+
+read-deployment-config:
+	$(eval POSTGRES_DATABASE_NAME="$(RESOURCE_NAME_PREFIX)-aytq-$(DEPLOY_ENV)${var.app_suffix}-psql-db")
+	$(eval POSTGRES_SERVER_NAME="$(RESOURCE_NAME_PREFIX)-aytq-$(DEPLOY_ENV)${var.app_suffix}-psql.postgres.database.azure.com")
+
+##@ Query parameter store to display environment variables. Requires Azure credentials
+set-azure-account: ${environment}
+	echo "Logging on to ${AZURE_SUBSCRIPTION}"
+	az account set -s ${AZURE_SUBSCRIPTION}
+
+.PHONY: install-fetch-config
+install-fetch-config: ## Install the fetch-config script, for viewing/editing secrets in Azure Key Vault
+	[ ! -f bin/fetch_config.rb ] \
+		&& curl -s https://raw.githubusercontent.com/DFE-Digital/bat-platform-building-blocks/master/scripts/fetch_config/fetch_config.rb -o bin/fetch_config.rb \
+		&& chmod +x bin/fetch_config.rb \
+		|| true
+
+edit-keyvault-secret: read-keyvault-config install-fetch-config set-azure-account
+	bin/fetch_config.rb -s azure-key-vault-secret:${KEY_VAULT_NAME}/${KEY_VAULT_SECRET_NAME} \
+		-e -d azure-key-vault-secret:${KEY_VAULT_NAME}/${KEY_VAULT_SECRET_NAME} -f yaml -c
+
+create-keyvault-secret: read-keyvault-config install-fetch-config set-azure-account
+	bin/fetch_config.rb -s azure-key-vault-secret:${KEY_VAULT_NAME}/${KEY_VAULT_SECRET_NAME} \
+		-i -e -d azure-key-vault-secret:${KEY_VAULT_NAME}/${KEY_VAULT_SECRET_NAME} -f yaml -c
+
+print-keyvault-secret: read-keyvault-config install-fetch-config set-azure-account
+	bin/fetch_config.rb -s azure-key-vault-secret:${KEY_VAULT_NAME}/${KEY_VAULT_SECRET_NAME} -f yaml
+
+validate-keyvault-secret: read-keyvault-config install-fetch-config set-azure-account
+	bin/fetch_config.rb -s azure-key-vault-secret:${KEY_VAULT_NAME}/${KEY_VAULT_SECRET_NAME} -d quiet \
+		&& echo Data in ${KEY_VAULT_NAME}/${KEY_VAULT_SECRET_NAME} looks valid
+
+terraform-init:
+	$(if $(IMAGE_TAG), , $(eval export IMAGE_TAG=main))
+	[[ "${SP_AUTH}" != "true" ]] && az account set -s $(AZURE_SUBSCRIPTION) || true
+	terraform -chdir=terraform init -backend-config workspace_variables/${DEPLOY_ENV}.backend.tfvars $(backend_config) -upgrade -reconfigure
```

Comment:
Should we standardise variable references to either ${} or $() where appropriate?

Reply:
done